api/bin/chainflip-lp-api/src/main.rs

```diff
@@ -455,10 +454,16 @@ impl RpcServer for RpcServerImpl {
 				if let Err(option_error) = order_fills(state_chain_client.clone(), block)
 					.await
 					.map_err(Some)
-					.and_then(|order_fills| match sink.send(&order_fills) {
-						Ok(true) => Ok(()),
-						Ok(false) => Err(None),
-						Err(error) => Err(Some(jsonrpsee::core::Error::ParseError(error))),
+					.and_then(|order_fills| {
+						if !order_fills.data.fills.is_empty() {
```

Comment:
No we want to leave this, and not fill when there are no fills.

Reply:
The reason I added this is because since we added blockhash+block number, the data that gets put into the data pipe is different every block. I've added this check here to prevent empty results from being returned every block. 
Is there a better way to prevent this spamming?

Reply:
We want to return empty results, when the block has no fills, it is no spamming, the lp's need this information, and we cannot change the behaviour of the existing rpcs that are being used.

Reply:
Remember this subscription already had the block hash and number, I think you are misunderstanding something.

Reply:
I see